style.css

```diff
@@ -1,3 +1,55 @@
 article p {
-    color: blue;
+    color: gold;
+    width: 50%;
+    text-align: justify;
+    
```

Comment:
Adding margin to the text elements and containers looks nice sometimes.
`margin: 10px;`